pyproject.toml

```diff
@@ -17,6 +17,7 @@ boto3 = "^1.28.23"
 pyopenssl = "^24.0.0"
 typing_extensions = "^4.7.1"
 jinja2 = "^3.1.2"
+python_hosts = "*"
```

Comment:
From MM: 

[I've researched it](https://github.com/jonhadfield/python-hosts/blob/devel/python_hosts/hosts.py). It's being well maintained for almost a decade, with recent updates. The src is pretty straightforward, and used in other operators (e.g. old kafka-broker charm). Can see no reason againts it. (c) @paulomach 

---

My call: today we do not have a written rule to pin major version.
It can be a new rule written by blood, but today it is not a blocker here.

Reply:
89a4e359